packages/framework/src/Resources/styles/admin/components/in/sort-arrows.less

```diff
@@ -19,9 +19,14 @@
 
         &--up {
             .draw-arrow-top(5px, @color-base, 5px, -8px);
+            position: absolute;
+            top: 14px;
```

Comment:
Should not be here `top: 0`? Or why is top position same as for down arrow?

Reply:
It is arrow position in column at table header. Arrow down is on same position but show another arrow.

Reply:
In this case it is okay. Thanks for explanation.